datadog/dogshell/wrap.py

```diff
@@ -289,11 +295,25 @@ def main():
     initialize(api_key=options.api_key)
     host = api._host_name
 
+    if options.warning_codes:
+        warning_codes = [int(w.strip()) for w in options.warning_codes.split(',')]
```

Comment:
You should write this as a function and use it as the type for your `add_option` call. It'll automatically be called when needed.